w3af/plugins/grep/analyze_cookies.py

```diff
@@ -204,13 +204,16 @@ def _http_only(self, request, response, cookie_obj,
         :return: None
         """
         if not self.HTTPONLY_RE.search(cookie_header_value):
-            
             vuln_severity = severity.MEDIUM if fingerprinted else severity.LOW
-            desc = 'A cookie without the HttpOnly flag was sent when ' \
+            keys = ''
+            for key in cookie_obj.keys(): 
+                keys += key + ' '
+            keys = keys.strip()
+            desc = 'Cookie(s) (%s) without the HttpOnly flag was sent when ' \
```

Comment:
I was thinking similarly along those lines, asking myself what if multiple cookies were sent with one HTTP response and only one of them being HttpOnly. I'll create a test for this. 

I'll change the description to match `Cookie "foobar" was sent without httponly"`